src/components/services/AirshipInstance.tsx

```diff
@@ -24,7 +24,7 @@ export function showError(error: unknown, options: ShowErrorWarningOptions = {})
     Bugsnag.notify(`showError: ${translatedError}`)
   }
   console.log(redText('Showing error drop-down alert: ' + makeErrorLog(error)))
-  Airship.show(bridge => <AlertDropdown bridge={bridge} message={translatedError} />)
+  Airship.show(bridge => <AlertDropdown bridge={bridge} message={translatedError} />).catch(err => console.error(err))
```

Comment:
Do we want the red box of doom in here?

Reply:
We really shouldn't be erroring here, so I'd argue this should be in devs faces if it were to ever fail.

Reply:
Fair enough.